examples/gno.land/r/gov/dao/types.gno

```diff
@@ -0,0 +1,232 @@
+package dao
+
+import (
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ufmt"
+)
+
+type ProposalID int64
+
+func (pid ProposalID) String() string {
+	return ufmt.Sprintf("%v", int64(pid))
+}
+
+// VoteOption is the limited voting option for a DAO proposal
+// New govDAOs can create their own VoteOptions if needed in the
+// future.
+type VoteOption string
+
+const (
+	AbstainVote VoteOption = "ABSTAIN" // Side is not chosen
+	YesVote     VoteOption = "YES"     // Proposal should be accepted
+	NoVote      VoteOption = "NO"      // Proposal should be rejected
+)
+
+type VoteRequest struct {
+	Option     VoteOption
+	ProposalID ProposalID
+	Metadata   Metadata
+}
+
+type ProposalRequestBuilder struct {
+	title       string
+	description string
+	callback    func(Metadata) error
+	metadata    Metadata
+}
+
+func NewProposalRequestBuilder(title string) *ProposalRequestBuilder {
+	return &ProposalRequestBuilder{
+		title: title,
+	}
+}
+
+func (prb *ProposalRequestBuilder) WithDescription(desc string) *ProposalRequestBuilder {
+	prb.description = desc
+	return prb
+}
+
+func (prb *ProposalRequestBuilder) WithMetadata(md Metadata) *ProposalRequestBuilder {
+	prb.metadata = md
+	return prb
+}
+
+func (prb *ProposalRequestBuilder) WithCallback(cb func(Metadata) error) *ProposalRequestBuilder {
+	prb.callback = cb
+	return prb
+}
+
+func (prb *ProposalRequestBuilder) Build() ProposalRequest {
+	return ProposalRequest{
+		title:       prb.title,
+		description: prb.description,
+		callback:    prb.callback,
+		metadata:    prb.metadata,
+	}
+}
+
+type ProposalRequest struct {
+	title       string
+	description string
+	callback    func(Metadata) error
+	metadata    Metadata
+}
+
+func (p *ProposalRequest) Title() string {
+	return p.title
+}
+
+func (p *ProposalRequest) Description() string {
+	return p.description
+}
+
+func (p *ProposalRequest) Callback() func(Metadata) error {
+	return p.callback
+}
+
+func (p *ProposalRequest) Metadata() Metadata {
+	return p.metadata
+}
+
+type Proposal struct {
+	author Author
+
+	title       string
+	description string
+
+	executor    *executor
+	allowedDAOs []string
+}
+
+func (p *Proposal) Author() Author {
+	return p.author
+}
+
+func (p *Proposal) Title() string {
+	return p.title
+}
+
+func (p *Proposal) Description() string {
+	return p.description
+}
+
+func (p *Proposal) ExecutorString() string {
+	if p.executor != nil {
+		return p.executor.String()
+	}
+
+	return ""
+}
```

Comment:
```suggestion
func (p *Proposal) Description() string {
	return p.description
}
```

This is where we should focus on defining useful description.

Reply:
It is done like that on purpose. How things are rendered to the user is part of the govDAO business logic. If we add any markdown or similar that joins both strings, we are limiting what can be done on future govDAO implementations.